scripts/script.js

```diff
@@ -200,3 +200,31 @@ for (let k = 0; k < projectsInfo.length; k += 1) {
     openPopup(k);
   });
 }
+
+// Validate form
+const form = document.getElementById('contact-form');
+const email = document.getElementById('useremail');
+const errorUpper = 'Your email is in uppercase. Please rewrite it in lowercase.';
+const small = document.createElement('small');
+
+function checkCase(input) {
+  return /[A-Z]/.test(input); // If has upper, return True
+}
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+
+  // Form invalid by default
+  let isValid = false;
+  const inputEmail = email.value.trim();
+  if (checkCase(inputEmail)) {
+    form.appendChild(small);
+    small.textContent = errorUpper;
+  } else {
+    form.removeChild(small);
+    isValid = true;
+  }
+  if (isValid) {
+    form.submit();
+  }
+});
```

Comment:
[OPTIONAL] I think you could wrap ALL of the JS code in a `main` function. And the call `main` once the web app is finished loading. This will ensure that it works correctly. Imagine that the script loads, but maybe some of the elements on the document aren't still ready. Then it could fail to select these.

Start the script running once the page has finished loading like this: `window.onload = main`

Reply:
I don't understand what you mean?

Do you mean I can write in my js file `function main() { _current contents of my js file_}`?
And then, in my index.js, write something like, `<body onload=main()>`?

Reply:
Or, now that I think about it, do you mean just this form validation funtion?

So I'd do a `function validate() { _my current validation code_ }` and then add a `form onload...>`?